listenbrainz/webserver/views/api.py

```diff
@@ -348,6 +349,58 @@ def validate_token():
         })
 
 
+@api_bp.route('/delete-listen', methods=['POST'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def delete_listen():
+    """ 
+    Delete a particular listen from the currently logged-in user's listen history.
```

Comment:
```suggestion
    Delete a particular listen from the a user's listen history.
```

This'll delete listens from whichever auth token it gets